app/views/groups/index.html.erb

```diff
@@ -9,10 +23,23 @@
 
   <tbody>
     <% @groups.each do |group| %>
-      <tr>
-        <td><%= group.id %></td>
+      <tr class="<%= group.neighborhood.id %> <%= group.kind %>">
```

Comment:
hmm @andylouisqin  I've never seen that error, and I dropped and remigrated several times. Let me try dropping and remigrating again to see if there's anything wrong.